lib/modules/hover.js

```diff
@@ -161,7 +161,6 @@ class Hover {
 	}
 
 	target(element: JQuery | HTMLElement) {
-		this.close();
```

Comment:
Removing this prevents you from hovering on something else while it's fading out:
![temp](https://user-images.githubusercontent.com/7673145/28759296-82fa7810-7569-11e7-82b5-b7529cc75456.gif)

(it's more noticeable when moving between multireddits)